src/config.rs

```diff
@@ -192,6 +194,12 @@ pub enum Config {
     ///
     /// See `crate::authres::update_authservid_candidates`.
     AuthservIdCandidates,
+
+    /// Let the core save all events to the database. You should expose this as an advanced
+    /// setting to the user. Enabling happens with posting a webxdc called `debug_logging` to
+    /// saved messages. When disabled, the logging webxdc gets deleted.
```

Comment:
Well you could use the checkbox to disable debug logging, but you are not able to enable it in the settings. That's what I meant but I'm not sure if it can nicely be reflected in the UIs.

Reply:
Well you could use the checkbox to disable debug logging, but you are not able to enable it in the settings. That's what I meant but I'm not sure if it can nicely be reflected in the UIs.